llm_benchmark_suite.sh

```diff
@@ -5,6 +5,9 @@ python llm_benchmark.py --minimal -b https://fixie-westus.openai.azure.com -m gp
 python llm_benchmark.py --minimal -b https://fixie-westus.openai.azure.com -m gpt-3.5-turbo-1106 "$@"
 python llm_benchmark.py --minimal -k $AZURE_EASTUS2_OPENAI_API_KEY -b https://fixie-openai-sub-with-gpt4.openai.azure.com -m gpt-4-1106-preview "$@"
 # python llm_benchmark.py --minimal -k $AZURE_EASTUS2_OPENAI_API_KEY -b https://fixie-openai-sub-with-gpt4.openai.azure.com -m gpt-3.5-turbo-1106 "$@"
+python llm_benchmark.py --minimal -k $AZURE_EASTUS2_LLAMA2_API_KEY -b https://fixie-llama-2-70b-serverless.eastus2.inference.ai.azure.com/v1 "$@"
+python llm_benchmark.py --minimal -k $AZURE_EASTUS2_MISTRAL_API_KEY -b https://fixie-mistral-serverless.eastus2.inference.ai.azure.com/v1 "$@"
```

Comment:
I'm surprised this works without adding "-m mistral-large" also since the model name is part of the request.

Reply:
I found that that wasn't necessary since the model seems to be embedded in the base_url, and in this case the model didn't need to be specified in the JSON body. Let me know if you saw something different.